models.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import Column, Integer, String, Date, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
+    __tablename__ = "authors"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False, unique=True)
```

Comment:
unique name? Is this because of task requirements or you've done it by yourself ?

Reply:
Yes, that condition were in a Readme. 
![image](https://github.com/mate-academy/py-fastapi-library-management-api/assets/92045507/29342c7a-293f-4a8e-8f3f-25622edde18c)